src/AppKit/NSCollectionLayoutAnchor.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using System.Runtime.InteropServices;
+using CoreFoundation;
+using Foundation;
+using ObjCRuntime;
+using CoreGraphics;
+
+namespace AppKit {
+
+	[Mac (10, 15)]
+	public enum NSCollectionLayoutAnchorOffsetType {
+		Absolute,
+		Fractional,
+	}
+
+	public partial class NSCollectionLayoutAnchor {
+		public static NSCollectionLayoutAnchor GetLayoutAnchor (NSDirectionalRectEdge edges, NSCollectionLayoutAnchorOffsetType offsetType, CGPoint offset) =>
```

Comment:
In general (I did not read the associated doc) when type 'X' has static members that returns 'X' then it's more a `Create` than a `Get`, e.g.

```csharp
public class Url {
   // good
   static Url Create (string url) { ... }
   // bad
   static Url Get (string url) { ... }

   // good
   static string GetDomain (Url url) { ... }
   // bad
   static string CreateDomain (Url url) { ... }
}
```